diff --git a/build/common.gypi b/build/common.gypi
index 1affc70..07b49c5 100644
--- a/build/common.gypi
+++ b/build/common.gypi
@@ -284,7 +284,7 @@
           ['OS=="win"', {
             'msvs_settings': {
               'VCCLCompilerTool': {
-                'RuntimeLibrary': '0',  # /MT.
+                'RuntimeLibrary': '2',  # /MT.
                 'Optimization': '3',
                 'AdditionalOptions': [
                   '/Zo',  # Improve debugging optimized builds.
@@ -315,7 +315,7 @@
           ['OS=="win"', {
             'msvs_settings': {
               'VCCLCompilerTool': {
-                'RuntimeLibrary': '1',  # /MTd.
+                'RuntimeLibrary': '3',  # /MTd.
                 'Optimization': '0',
               },
             },
